### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by lemastero**
# Description

Update library dependencies.
(Current versions of monix and cats have some issues with diverging implicits so they are not updated.)

# Proposed Solution

# Important Changes Introduced

# Testing

Tested by: `sbt clean test`

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -16,8 +16,8 @@ val commonSettings = Seq(
 val malletDeps = Seq(
   Dependencies.scopt
 ).flatten ++ Seq(
-  "org.jline" % "jline" % "3.1.2",
-  "net.java.dev.jna" % "jna" % "4.5.1"
+  "org.jline" % "jline" % "3.16.0",
```
Comment (kapke) [inline build.sbt:19]: using the occasion: shouldn't jline and jna be picked from `project/Dependencies.scala` as well?

**Review by kapke — COMMENTED** [review]
Remember to update `repo.nix` files as well, these are like lockfiles from other package managers, but required (and managed) by nix to have build reproducible.
To do that:
  - install nix (may get tricky on macOS Catalina 😅 )
  - enter nix shell (`nix-shell`)
  - run `sbtix-gen-all2` command

File: build.sbt (hunk 1 above)
Comment (lemastero) [reply build.sbt]: Moved to Dependencies.

**Review by kapke — APPROVED** [review]
LGTM!
Since this touches metrics as well, I think @mmrozek should take a look too.

**Comment by mmrozek** [comment]
@lemastero you have to sign all commits

**Comment by lemastero** [comment]
Could not sign old commits. I have created different PR: #692

**Pull request closed without merging**